src/System.Management.Automation/engine/hostifaces/InternalHostUserInterface.cs

```diff
@@ -88,7 +88,7 @@ public override bool SupportsVirtualTerminal
         /// </summary>
         /// <exception cref="HostException">
         /// if the UI property of the external host is null, possibly because the PSHostUserInterface is not
-        ///     implemented by the external host
+        /// implemented by the external host
```

Comment:
```suggestion
        /// implemented by the external host.
```

Reply:
fixed in 198b33a4a8ee1e69fdb23b1a242458cafdb5fb7a